### DISCUSSION TITLE
Using a fontsize less that 4pt in add_redact_annot

### DISCUSSION THREAD

**Discussion by francesco-bellei** [pymupdf/PyMuPDF#3595] [Looking for help]
Hi everyone!
I was reading the documentation of Page.add_redact_annot() and I read that for what regards the parameter fontsize:

fontsize (float) – the fontsize to use for the replacing text. If the text is too large to fit, several insertion attempts will be made, gradually reducing the fontsize to no less than 4. If then the text will still not fit, no text insertion will take place at all.

Is there any way to overcome this limit and put inside an annotation a smaller text? Like with fontsize 2 ?
Thanks in advance.

**Comment by JorjMcKie** [comment]
I suggest to use `page.insert_htmlbox()` to fill the rectangle. This will shrink to replacement text with no lower limit.
